server/lib/fbApi.js

```diff
@@ -444,6 +444,19 @@ const del = ({canonical = null} = {}) => {
 const wipe = () => list({fields: ['id']})
 .then(items => Promise.all(items.map(item => del({id: item.id}))));
 
+const posts = ({since} = {}) => call(
```

Comment:
Scratch that. I think we also need to filter out any posts with `is_published = false` or `is_hidden = false`. See https://developers.facebook.com/docs/graph-api/reference/post/#read.